app/views/account/tenant/edit.html.slim

```diff
@@ -0,0 +1,4 @@
+h1.col-md-6.col-md-offset-3 Edit Tenant
+.row
+  .col-md-3.col-md-offset-3
+  	= render 'form', button_text: "Save Tenant", form_path: account_tenant_path
```

Comment:
invalid slim formatting for this row

stop using tabs, use spaces

Reply:
Done.